tests/render/fixtures/ValidationResultsPageRenderer_render_with_run_info_at_end.json

```diff
@@ -2410,70 +2370,30 @@
                       ]
                     }
                   },
-                  "table": [
-                    [
-                      22.0
-                    ],
-                    [
-                      21.0
-                    ],
-                    [
-                      30.0
-                    ],
-                    [
-                      18.0
-                    ],
-                    [
-                      36.0
-                    ],
-                    [
-                      24.0
-                    ],
-                    [
-                      26.0
-                    ],
-                    [
-                      27.0
-                    ],
-                    [
-                      28.0
-                    ],
-                    [
-                      19.0
-                    ],
-                    [
-                      20.0
-                    ],
-                    [
-                      23.0
-                    ],
-                    [
-                      25.0
-                    ],
-                    [
-                      32.0
-                    ],
-                    [
-                      45.0
-                    ],
-                    [
-                      29.0
-                    ],
-                    [
-                      31.0
-                    ],
-                    [
-                      33.0
-                    ],
-                    [
-                      35.0
-                    ],
-                    [
-                      39.0
-                    ]
+                  "table":[
+                      [22.0, 35],
+                      [21.0, 31],
+                      [30.0, 31],
+                      [18.0, 30],
+                      [36.0, 29],
+                      [24.0, 27],
+                      [26.0, 27],
+                      [27.0, 24],
+                      [28.0, 24],
+                      [19.0, 23],
+                      [20.0, 23],
+                      [23.0, 23],
+                      [25.0, 23],
+                      [32.0, 22],
+                      [45.0, 20],
+                      [29.0, 15],
+                      [31.0, 14],
+                      [33.0, 14],
+                      [35.0, 14],
+                      [39.0, 14]
                   ],
                   "header_row": [
-                    "Sampled Unexpected Values"
+                    "Sampled Unexpected Values", "Count"
```

Comment:
This is probably the most illustrative example : We are able to display the top-20 most common unexpected values and their counts (which are clearly across the entire dataset), but we were not.